src/main/kotlin/no/nav/familie/ef/mottak/service/OppgaveService.kt

```diff
@@ -45,12 +45,25 @@ class OppgaveService(private val integrasjonerClient: IntegrasjonerClient,
 
     fun lagBehandleSakOppgave(journalpost: Journalpost, behandlesAvApplikasjon: String): Long {
         val opprettOppgave = opprettOppgaveMapper.toBehandleSakOppgave(journalpost, behandlesAvApplikasjon)
-        val nyOppgave = integrasjonerClient.lagOppgave(opprettOppgave)
+        try {
+            val nyOppgave = integrasjonerClient.lagOppgave(opprettOppgave)
+            log.info("Oppretter ny behandle-sak-oppgave med oppgaveId=${nyOppgave.oppgaveId} " +
+                     "for journalpost journalpostId=${journalpost.journalpostId}")
+            return nyOppgave.oppgaveId
+        } catch (httpStatusCodeException: HttpStatusCodeException) {
+            if (finnerIngenGyldigArbeidsfordelingsenhetForBruker(httpStatusCodeException)) {
+                val oppgaveId = integrasjonerClient.lagOppgave(opprettOppgave.copy(enhetsnummer = ENHETSNUMMER_NAY)).oppgaveId
+                log.warn("Oppgave kunne ikke finne enhet for behandle-sak-oppgave med oppgaveId=${oppgaveId} "
+                         + "for journalpost journalpostId=${journalpost.journalpostId}. Bruker ENHETSNUMMER_NAY")
+                return oppgaveId
+            } else {
+                log.error("Oppgave kunne ikke finne enhet for behandle-sak-oppgave  "
+                          + "for journalpost journalpostId=${journalpost.journalpostId}. Bruker ENHETSNUMMER_NAY")
 
-        log.info("Oppretter ny behandle-sak-oppgave med oppgaveId=${nyOppgave.oppgaveId} " +
-                 "for journalpost journalpostId=${journalpost.journalpostId}")
+                throw httpStatusCodeException
+            }
```

Comment:
Finner vi et bra metodenavn for denne bolken? Eller kan vi gjenbruke `opprettOppgaveMedEnhetFraNorgEllerBrukNayHvisEnhetIkkeFinnes`, men logge verdiene mer generisk fra Oppgave:
 `oppgave.oppgavetype og oppgave.journalpostId` ?

Reply:
La til en test og refaktorerte litt. Takk for tilbakemelding :)